src/activities/bargame/Bargame.qml

```diff
@@ -0,0 +1,395 @@
+
+        DialogActivityConfig {
+            id: dialogActivityConfig
+            currentActivity: activity
+            content: Component {
+                Item {
+                    property alias modeBox: modeBox
+
+                    property var availableModes: [
+                        { "text": qsTr("Easy"), "value": 1 },
+                        { "text": qsTr("Medium"), "value": 2 },
+                        { "text": qsTr("Difficult"), "value": 3 }
+                    ]
+                    Flow {
+                        id: flow
+                        spacing: 5
+                        width: dialogActivityConfig.width
+                        GCComboBox {
+                            id: modeBox
+                            model: availableModes
+                            background: dialogActivityConfig
+                            label: qsTr("Select your mode")
+                        }
+                    }
+                }
+            }
+            onClose: {
+                Activity.initLevel();
+                home();
+            }
+            onLoadData: {
+                if(dataToSave && dataToSave["mode"]) {
+                    items.mode = dataToSave["mode"];
+                    Activity.initLevel();
+                }
+            }
+            onSaveData: {
+                var newMode = dialogActivityConfig.configItem.availableModes[dialogActivityConfig.configItem.modeBox.currentIndex].value;
+                if (newMode !== items.mode) {
+                    items.mode = newMode;
+                    dataToSave = {"mode": items.mode};
+                }
+            }
+            function setDefaultValues() {
+                for(var i = 0 ; i < dialogActivityConfig.configItem.availableModes.length ; i++) {
+                    if(dialogActivityConfig.configItem.availableModes[i].value === items.mode) {
+                        dialogActivityConfig.configItem.modeBox.currentIndex = i;
+                        break;
+                    }
+                }
+            }
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level | config }
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+            onConfigClicked: {
+                dialogActivityConfig.active = true
+                // Set default values
+                dialogActivityConfig.setDefaultValues();
+                displayDialog(dialogActivityConfig)
+            }
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: {
+                win.connect(Activity.initLevel)
+                loose.connect(Activity.initLevel)
+            }
+        }
+
+        Score {
+            id: score
+            message: "Score: " + items.score
```

Comment:
use qsTr(). But for this, I would use 2 separate numbers, one for your score, the other for the opponent instead of having only one number that is incremented on win an decremented on lose.